packages/shared/src/components/auth/OnboardingSteps/ContentTypes/useContentTypesOnboarding.tsx

```diff
@@ -0,0 +1,36 @@
+// TODO: move in right directory if the onboardingContentType experiment is successful
```

Comment:
What would be the right directory?

Reply:
Well this is a good place, unless we'll use this in other places in the future where hooks directory is the best place. 
Thinking this doesn't have to be a hook really, just a utils function for content type should be enough

Reply:
removed the hook at all, felt unnecessary, instead created a helper function for separating functionality